io-sim/src/Control/Monad/IOSim/Internal.hs

```diff
@@ -198,7 +198,8 @@ schedule !thread@Thread{
   assert (invariant (Just thread) simstate) $
   case action of
 
-    Return x -> case ctl of
+    Return x -> {-# SCC "schedule.Return" #-}
```

Comment:
Is this necessary to stay? (The same for all changes in this commit 1d5d3763e94b5a16b9e2177b5d40351d5157079b)

Reply:
It doesn't cost us to keep it, and might prove to be useful in the future.